content/master/whats-crossplane/_index.md

```diff
@@ -223,6 +224,62 @@ GCP, Terraform, Helm, GitHub, etc to support Crossplane v2 soon.
 <!-- vale gitlab.FutureTense = YES -->
 {{</hint>}}
 
+### Operations
+
+Operations let you run operational tasks using function pipelines.
+
+While composition and managed resources focus on creating and managing
+infrastructure, operations handle tasks that don't fit the typical resource
+creation pattern - like certificate monitoring, rolling upgrades, or scheduled
+maintenance.
+
+**Operations run function pipelines to completion like a Kubernetes Job.**
+Instead of continuously managing resources, they perform specific tasks and
+report the results.
+
+```mermaid
+flowchart TD
+user(User)
+
+subgraph control [Control Plane]
+  operation(SSL Monitor Operation API)
```

Comment:
i'm seeing this getting clipped to `SSL Monitor Operation AP`:
<img width="229" height="66" alt="Screenshot 2025-08-07 at 4 45 20 PM" src="https://github.com/user-attachments/assets/6afee99e-086d-4d57-b36f-bd84fbb25136" />